roles/get_bmc_facts/tasks/main.yml

```diff
@@ -16,5 +16,10 @@
 
 - name: Extract all FW versions to facts
   ansible.builtin.set_fact:
-    "{{ 'get_bmc_facts_' + item['Id'] | lower }}": "{{ item['Version'] }}"
+    bmc_fw_version: "{{ bmc_fw_version | default({}) | combine({ item['Id'] | lower : item['Version'] }) }}"
   loop: "{{ result.redfish_facts.firmware.entries }}"
```

Comment:
`bmc_fw_version` looks like wrong var name... we have here all the inventory, no ?

Reply:
I agree it is a beyond confusing variable name ... I was using version as in "Version" from redfish ... Fact also has a bigger name in ansible; this is just a part of the facts ... If we cannot come up with something better, I'd be happy to move it back to "fact"

Reply:
how about `get_bmc_facts_all_fw_versions` ?